### DISCUSSION TITLE
Is there a way to save settings permanently to the controller?

### DISCUSSION THREAD

**Discussion by Mareck** [devcompl/TTController#207] [Q&A]
I have two Riing RGB controllers with 9 fans controlled by TT RGB plus app 2.0.7. After reboot, the fans lights go back to the stupid rainbow and speed go back to the default value. TT RGB Plus_Setup_1.1.7 can save the settings permanently to the controller but this version of the app doesn't recognize the newer version of the controllers.

Is there  a way to save settings permanently to the controller box using TTController service?

![Thermaltake-rgb-controller jpg 2bd2cd55794d0de8101d7f597cd3b888](https://user-images.githubusercontent.com/45420741/216762227-1ea94664-391f-4c6e-8537-9c44d4fb2999.jpg)
![TT_RGBplus-controller2 jpg 35449f0b8c07b2ebd686a33020715c09](https://user-images.githubusercontent.com/45420741/216762380-3aa7bd7a-3418-4a91-b91c-5383ebc95c1f.jpg)

**Comment by MoshiMoshi0** [comment]
Riing Quad controllers default to rainbow if they don't receive any data and don't allow saving.
Unless this is some newer controller.
